lab-materials/05/05-05/run_process_claims_pipeline.py

```diff
@@ -11,10 +11,15 @@
 
 from process_claims_pipeline import process_claims_pipeline
 
-COMPILE=False
+COMPILE=True
+
+metadata = {
+        "claim_ids": 0,
+        "detection_endpoint": "https://some-endpoint",
```

Comment:
@RHRolun is this expected to be this url, or it's just a mockup and any variable will work?

Reply:
Mockup here, the users will enter the url to their served model when running the pipeline (this becomes default parameters which suggest the user what to enter)